app/src/main/java/com/main/mealplanner/service/RecipeService.kt

```diff
@@ -20,24 +20,15 @@ class RecipeService {
         val service = RetrofitClientInstance.retrofitInstance?.create(IRecipeDAO::class.java)
         val call = service?.getAllRecipes()
         call?.enqueue(object: Callback<RecipeList> {
-            /**
-             * Invoked for a received HTTP response.
-             *
-             *
-             * Note: An HTTP response may still indicate an application-level failure such as a 404 or 500.
-             * Call [Response.isSuccessful] to determine if the response indicates success.
-             */
```

Comment:
Why remove this?  It looks fairly descriptive.

Reply:
I opted to do this to make it make it less clutter.